docker-compose/scripts/run-dev.sh

```diff
@@ -8,11 +8,23 @@ fi
 
 elastic_host=${kuzzle_services__storageEngine__client__node:-http://elasticsearch:9200}
 
+if [ ! -d "./node_modules/" ]; then
+    git submodule init
+    git submodule update
+    npm ci --unsafe-perm
+    ./docker-compose/scripts/install-plugins.sh
+fi
```

Comment:
These lines seem redundant: the `elif` condition below seems sufficient. What's their added value?

Reply:
Yep :+1: , test/old chunk went through. Fixed